js/charts/clusteredverticalbar.js

```diff
@@ -110,84 +121,120 @@ function clusteredverticalbar() {
         this.fontSize(config.fontSize);
         this.isFilterGrid(config.isFilterGrid);
         this.showSorting(config.showSorting);
-        this.displayColorExpression(config.displayColorExpression)
-        this.textColorExpression(config.textColorExpression)
+        this.displayColorExpression(config.displayColorExpression);
+        this.textColorExpression(config.textColorExpression);
         setDefaultColorForChart();
-        this.legendData(_displayColor, config.measure, config.displayNameForMeasure);
-    }
+        this.legendData(
+            _displayColor,
+            config.measure,
+            config.displayNameForMeasure
+        );
+    };
 
     var setDefaultColorForChart = function () {
         for (let index = 0; index < _measure.length; index++) {
-            if (_displayColor[index] == null || _displayColor[index] == undefined) {
+            if (
+                _displayColor[index] == null ||
```

Comment:
only if(_displayColor[index] ) is enough?

Reply:
done